src/Main.purs

```diff
@@ -112,7 +118,11 @@ main cliRoot = do
     Left err ->
       Console.error $ ArgParser.printArgError err
     Right (Build args) ->
-      launchAff_ $ runBuild args
+      flip runAff_ (runBuild args) case _ of
+        Right _ -> pure unit
+        Left err -> do
+          Console.error (message err)
+          Process.exit 1
```

Comment:
Any nicer way to write this?

Reply:
I think this is the boring but readable way to write this 😄

Reply:
Technically, we should be using `Process.setExitCode 1` because `Process.exit 1` could cause content not to be flushed to `stdout`/`stderr` before the Node process exits. See https://nodejs.org/dist/latest-v18.x/docs/api/process.html#processexitcode

Reply:
Our version of `node-process` doesn't have that function. I'll fix all these in a separate PR. Thanks!